section_faiss_vector_index.go

```diff
@@ -619,13 +557,19 @@ func (vo *vectorIndexOpaque) process(field index.VectorField, fieldID uint16, do
 
 // todo: better hash function?
 // keep the perf aspects in mind with respect to the hash function.
-// random seed based hash golang.
+// Uses a time based seed to avoid 2 identical vectors in different
+// segments having the same hash -> could be an issue when merging those
+// segments
```

Comment:
If you're not going to be de-duplicating vectors from different docIDs - why would there be an issue?
Also - you cannot leave comments like these in here, address them!

Reply:
If you're referring to the merge 'issue' in the comment as something to be addressed, it's already addressed when ensuring that no identical vectors have identical hashes. What I meant was that identical vectors having identical hashes was to be avoided during merge.
I think I didn't phrase my comment right.